src/services/LogFileManager/decodeUtils.ts

```diff
@@ -78,7 +78,6 @@ const tryCreateDecoderBySignature = async (
                 };
             } catch (e) {
                 console.warn(`Magic number matches ${entry.name}, but decoder creation failed:`, e);
-                break;
```

Comment:
discussed offline - the removal is needed so that all decoders with matching signatures can be tried